runtime/doc/lua.txt

```diff
@@ -2359,33 +2359,49 @@ find({names}, {opts})                                          *vim.fs.find()*
     The search can be narrowed to find only files or only directories by
     specifying {type} to be "file" or "directory", respectively.
 
-    If {names} is a function, it is called for each file and directory within
-    the traversed directory and it would be passed the base name of each file
-    or directory and the path of the directory being traversed. The function
-    should return `true` if the given file or directory is considered a match.
+    Examples: >lua
+
+     -- location of Cargo.toml
+     local cargo = vim.fs.find(
```

Comment:
I think this example should include the `path` key since that is necessary for getting the correct LSP root directory (the main motivation for this example , and in fact this function).

Here's the function I use in my config (`file` is `args.file` from a `FileType` autocommand):
```lua
local find_root = function(markers, file)
  return vim.fs.dirname(vim.fs.find(markers, { upward = true, path = vim.fn.fnamemodify(file, ':p:h') })[1])
end
```